src/cmdlib.sh

```diff
@@ -375,15 +375,22 @@ prepare_compose_overlays() {
         exit 1
     fi
 
-    if [ -d "${overridesdir}" ] || [ -d "${ovld}" ]; then
+    if [ -d "${overridesdir}" ] || [ -d "${ovld}" ] || [ -d "${workdir}/src/yumrepos" ]; then
         mkdir -p "${tmp_overridesdir}"
         cat > "${override_manifest}" <<EOF
 include: ${manifest}
 EOF
         # Because right now rpm-ostree doesn't look for .repo files in
         # each included dir.
         # https://github.com/projectatomic/rpm-ostree/issues/1628
-        cp "${workdir}"/src/config/*.repo "${tmp_overridesdir}"/
+        find "${configdir}" -name '*.repo' -exec cp -t "${tmp_overridesdir}" {} +
+        if [ -d "${workdir}/src/yumrepos" ]; then
+            find "${workdir}/src/yumrepos" -name '*.repo' -exec cp -t "${tmp_overridesdir}" {} +
+        fi
+        if ! ls "${tmp_overridesdir}"/*.repo; then
```

Comment:
I think the issue is here as this assumes that there is an overridesdir when there may not be

Reply:
Where this code is, we always create an overrides dir. ISTM like the issue OpenShift CI is hitting is the same that FCOS was hitting and *should* be fixed by [`24ffe87` (#3045)](https://github.com/coreos/coreos-assembler/pull/3045/commits/24ffe87937b915b2939ca235cf541731c8cf1289). Let's see what the CI rerun in https://github.com/openshift/os/pull/959 says. :)